File: OCH_Tao/ABC/A/337.py
N = int(input())
T = 0
A = 0
for i in range(N):
  x,y = map(int,input().split())
  T += x
  A += y
if T>A:
  print("Takahashi")
elif T==A:
  print("Draw")
else:
  print("Aoki")

File: OCH_Tao/ABC/A/337.js
const [, ...input] = (require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"));
let T = 0;
let A = 0;
for (const i of input) {
  const [X, Y] = i.split(" ").map(x => parseInt(x, 10));
  T += X;
  A += Y;
}
console.log(T > A ? "Takahashi" : T < A ? "Aoki" : "Draw");